### DISCUSSION TITLE
Please add Debian/Ubuntu repo for 23.04 (lunar)

### DISCUSSION THREAD

**Discussion by pohutukawa** [aquasecurity/trivy#4340] [Ideas]
### Description

The current install instructions request to add a repo for the codename release of Ubuntu. However unfortunately there is none for the current stable Ubuntu 23.04 (lunar) release, and one has to default back to `kinetic`.

It shouldn't be difficult to resolve this for packaging.

### Target

None

### Scanner

None

**Comment by knqyf263** [comment]
Thanks. There is an issue. Please track it.
https://github.com/aquasecurity/trivy/issues/4298
